spatialmath/capsule.go

```diff
@@ -269,7 +269,13 @@ func capsuleVsMeshDistance(c *capsule, other *Mesh) float64 {
 	lowDist := math.Inf(1)
 	for _, t := range other.triangles {
 		// Measure distance to each mesh triangle
-		dist := capsuleVsTriangleDistance(c, t)
+		// Make sure the triangle is transformed by the pose of the mesh to ensure that it is properly positioned
+		properlyPositionedTriangle := NewTriangle(
```

Comment:
Capsule-box distance calculation relies on this *not* happening to be correct. That will need to be updated as well.

Consider making this a Transform method on Triangle.